client/src/components/PinkHeader/PinkHeader.css

```diff
@@ -0,0 +1,63 @@
+*{
+    font-family: var(--default-font)
+}
+
+.PinkHeader {
+    background-image: var(--profile-banner);
+    background-repeat:no-repeat;
+    background-size: cover;
+    width: 1440px;
+    height: 231px;
```

Comment:
Does this need an explicit height? Is there a way we can add a consistent amount of padding? If the content in the header is too big, this will break it